content/en/integrations/faq/client-authentication-against-the-apiserver-and-kubelet.md

```diff
@@ -28,5 +28,16 @@ The default for kubelet traffic is to first try to use the read-only port that d
 kubelet_tls_verify: True
 ```
 
+## Alternate Option: kubeconfig
+
+Alternately, you can also use [kubeconfig][3] for API server authentication. Use the `DD_KUBERNETES_KUBECONFIG_PATH` environment variable to specify the path, or the equivalent option in [datadog.yaml][4]
+
+```
+kubernetes_kubeconfig_path: /path/to/file
+```
+
+
 [1]: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod
 [2]: https://kubernetes.io/docs/concepts/configuration/secret
+[3]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig
+[4]: https://github.com/DataDog/datadog-agent/blob/a9e0c4f534482170817300edb3cce01df9abea4a/pkg/config/config_template.yaml#L687-L692
```

Comment:
Should we link to a static version of `datadog.yaml`?

Reply:
i think config_template is best actually